content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing ] parts of code to verify its correctness.'
```

Comment:
```suggestion
Description: 'Unit testing involves testing parts of code to verify its correctness.'
```

Reply:
Maybe instead of correctness, we can say behavior? Hopefully can get another eye on this as well ☺️

LMK what you think!

Reply:
I think behavior works too, I'll use this instead.

Reply:
```suggestion
Description: 'Unit testing involves testing ] parts of code to verify its behavior.'
```